examples/odata-cli/src/activities.ts

```diff
@@ -5,9 +5,11 @@ import {
     ODataVersion,
     TransportChecksService,
     TransportRequestService,
-    ListPackageService
+    ListPackageService,
+    FileStoreService
 } from '@sap-ux/axios-extension';
 import { logger } from './types';
+import type { ArchiveFileNode } from '@sap-ux/axios-extension/src/abap/types';
```

Comment:
Why import from `@sap-ux/axios-extension/src/abap/types`?

Reply:
Fixed by exposing the type.